### DISCUSSION TITLE
Request: Opening book starts playing automatically

### DISCUSSION THREAD

**Discussion by intrepidclass** [Goodwy/PlayBooks#70] [Ideas]
99 times out of a hundred, when I open a book, I want it to start playing. I would prefer not to have to click the play button too to achieve this.

I would be fine with this being an option you have to enable in the settings. Though I wonder what's the reasoning behind it not already being like this? What are people doing opening books and not wanting it to play right away? :D

**Comment by Goodwy** [comment]
Hi. Good idea, we will add it in the next versions.

**Reply by devvythelopper** [reply]
> Hi. Good idea, we will add it in the next versions.

But please allow the user to enable/disable that behaviour. I for one am quite happy that the books do NOT start playing when I click on them. I sometimes just want to get information about a book or access my bookmarks and not get the player started...

**Comment by intrepidclass** [comment]
Awesome!
